kotlinlib/src/mill/kotlinlib/detekt/DetektModule.scala

```diff
@@ -21,7 +21,7 @@ trait DetektModule extends KotlinModule {
 
   private def detekt0() = T.task {
 
-    val args = detektOptions() ++ Seq("-i", PathRef(T.workspace).path.toString()) ++
```

Comment:
`PathRef(T.workspace)` is calculating an expensive (but unused) hash doomed to change as soon as we finish the task and write the cache to disc (into `T.workspace/"out"`).